src/frameName.cpp

```diff
@@ -177,6 +194,14 @@ void FrameName::javaMethodName(jmethodID method) {
         (err = jvmti->GetClassSignature(method_class, &class_name, NULL)) == 0) {
         // Trim 'L' and ';' off the class descriptor like 'Ljava/lang/Object;'
         javaClassName(class_name + 1, strlen(class_name) - 2, _style);
+        if (_includemm) {
+            jvmti->GetMethodModifiers(method, &modifiers);
+            for (std::map<int, std::string>::const_iterator iterator = access_flags.begin(); iterator != access_flags.end(); iterator++) {
```

Comment:
Will iterating over the flags that way ensure we get them in the "natural" order, i.e the order they are usually written in files? For example, not `abstract final private myMethod` but `private final abstract myMethod`

Reply:
It would check from lowest value of key to highest which I though was "good" order, but I checked that "good practice" order is a little bit different [here](https://checkstyle.sourceforge.io/config_modifier.html#:~:text=TreeWalker-,ModifierOrder,-Since%20Checkstyle%203.0 ) so I changed the container to vector

Reply:
Cool. You can give the link https://checkstyle.sourceforge.io/config_modifier.html#ModifierOrder as a comment there as well, to describe the order you chose :)